syntax = "proto3";

package gamer;

service Gamer {
    rpc Ping(PingMessage) returns (PingMessage) {}
    rpc Action(ActionInput) returns (ActionOutput) {}
}

message PingMessage {
    string id = 1;
}

message ActionInput {
    string id = 1;
    string gameId = 2;
    int32 totalRound = 3;
    int32 currentRound = 4;
    repeated string data = 5;
}

message ActionOutput {
    string id = 1;
    repeated string data = 2;
}
